examples/example.py

```diff
@@ -6,6 +6,9 @@
 def main():
     # make input data
     site = PVSite(latitude=51.75, longitude=-1.25, capacity_kwp=1.25)
+    # for enphase testing
```

Comment:
Think it's fine to leave the comment like this here but might be cleaner just to remove it for now and then we can add it in after it's been tested or even create a new example file with the inverter_type used

Reply:
Yeah that sounds good, did it in my latest commit